react/src/components/dropdown/SprkDropdown.test.js

```diff
@@ -228,3 +228,41 @@ it('should not set href if the supplied tagname is not a', () => {
       .getAttribute('href'),
   ).toBe(null);
 });
+
+it('should choose correct aria-label text if default variant', () => {
```

Comment:
should test for when title or screenReaderText or triggerText arent supplied also